### PR TITLE
feat(plasma-ui): Add ref for `TabItem` component

### PR DESCRIPTION
**Pull request by neretin-trike**
:baby_chick: Download canary assets:

[colors--canary.209.3381016224.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.209.3381016224.xml)  
[color_sbermarket_ios-swift--canary.209.3381016224.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.209.3381016224.swift)  
[color_sbermarket_kotlin--canary.209.3381016224.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.209.3381016224.kt)  
[color_sbermarket_react-native--canary.209.3381016224.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.209.3381016224.ts)  
[color_sberprime_ios-swift--canary.209.3381016224.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.209.3381016224.swift)  
[color_sberprime_kotlin--canary.209.3381016224.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.209.3381016224.kt)  
[color_sberprime_react-native--canary.209.3381016224.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.209.3381016224.ts)  
[PlasmaTokensColor--canary.209.3381016224.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.209.3381016224.swift)  
[typo_mage_kotlin--canary.209.3381016224.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.209.3381016224.kt)  
[typo_mage_react-native--canary.209.3381016224.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.209.3381016224.ts)  
[typo_plasma_kotlin--canary.209.3381016224.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.209.3381016224.kt)  
[typo_plasma_react-native--canary.209.3381016224.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.209.3381016224.ts)  
[typo_ruler_kotlin--canary.209.3381016224.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.209.3381016224.kt)  
[typo_ruler_react-native--canary.209.3381016224.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.209.3381016224.ts)  
[typo_sage_kotlin--canary.209.3381016224.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.209.3381016224.kt)  
[typo_sage_react-native--canary.209.3381016224.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.209.3381016224.ts)  
[typo_sbermarket_kotlin--canary.209.3381016224.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.209.3381016224.kt)  
[typo_sbermarket_react-native--canary.209.3381016224.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.209.3381016224.ts)  
[typo_soulmate_kotlin--canary.209.3381016224.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.209.3381016224.kt)  
[typo_soulmate_react-native--canary.209.3381016224.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.209.3381016224.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-temple@1.109.0-canary.209.3381016224.0
  npm install @salutejs/plasma-ui@1.143.0-canary.209.3381016224.0
  # or 
  yarn add @salutejs/plasma-temple@1.109.0-canary.209.3381016224.0
  yarn add @salutejs/plasma-ui@1.143.0-canary.209.3381016224.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-209/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-209/
  ui storybook: http://plasma.sberdevices.ru/pr-209/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-209/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-209/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-209/temple-storybook/

**Review by Yeti-or — APPROVED** [review]

File: packages/plasma-ui/src/components/Tabs/TabItem.tsx [hunk 1]
```diff
@@ -81,14 +81,19 @@ export const StyledTabItemMemo = React.memo(StyledTabItem);
 /**
  * Элемент списка вкладок, недопустимо использовать вне компонента Tabs.
  */
-export const TabItem = (props: TabItemProps) => {
-    const ref = useRef<HTMLElement>(null);
+/**
+ * Элемент списка вкладо`к, недопустимо использовать вне компонента Tabs.
```
Comment (Yakutoc) [inline packages/plasma-ui/src/components/Tabs/TabItem.tsx:85]: опечатка - `вкладо`к,`

**Review by Yakutoc — APPROVED** [review]

File: packages/plasma-ui/src/components/Tabs/TabItem.tsx (hunk 1 above)
Comment (neretin-trike) [reply packages/plasma-ui/src/components/Tabs/TabItem.tsx:85]: поправлю, спасиб

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-209/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-209/
  ui storybook: http://plasma.sberdevices.ru/pr-209/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-209/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-209/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-209/temple-storybook/

**Pull request merged**